pkg/config/ovn/ovn.go

```diff
@@ -77,3 +130,23 @@ func NewOVNKubernetesConfigFromFileOrDefault(path string) (*OVNKubernetesConfig,
 	}
 	return nil, fmt.Errorf("getting OVNKubernetes config: %v", err)
 }
+
+func GetOVNGatewayIP() (string, error) {
+	iface, err := net.InterfaceByName(OVNGatewayInterface)
+	if err != nil {
+		return "", err
+	}
+	addrs, err := iface.Addrs()
+	if err != nil {
+		return "", err
+	}
+	for _, addr := range addrs {
+		ip := addr.(*net.IPNet).IP
+		if ip.To4() != nil {
```

Comment:
Do we need `ip.To4()`? Looks like `ip.String()` already does that under the hood: https://cs.opensource.google/go/go/+/refs/tags/go1.19.4:src/net/ip.go;l=321